1lab.py

```diff
@@ -0,0 +1,34 @@
+class DogCollar:
+    dogBreed = "Every breed"
+
+    def __init__(self, brend  = 'Unkown', maxLenths = 'Unkown', minLenths = 'Unkown', color = 'Unkown'):
```

Comment:
[PEP8](https://www.python.org/dev/peps/pep-0008/) тут і далі

```suggestion
    def __init__(self, brend='Unkown', max_length='Unkown', min_length='Unkown', color='Unkown'):
```